src/amd/compiler/aco_instruction_selection.cpp

```diff
@@ -785,10 +807,6 @@ void visit_alu_instr(isel_context *ctx, nir_alu_instr *instr)
          emit_sop2_instruction(ctx, instr, aco_opcode::s_add_i32, dst, true);
          break;
       }
-      if (dst.regClass() == v1 && ctx->options->chip_class >= GFX9) {
-         emit_vop2_instruction(ctx, instr, aco_opcode::v_add_u32, dst, true);
-         break;
-      }
       Temp src0 = get_alu_src(ctx, instr->src[0]);
       Temp src1 = get_alu_src(ctx, instr->src[1]);
       if (src1.type() == sgpr) {
```

Comment:
I think, here we check already if src1 is sgpr. guess, we can remove this portion then.